File: pasukhi-admin/src/types/admin-user.ts
export type ManagedAdminUser = {
  id: string
  email: string
  firstName: string
  lastName: string
  role: string
  businessId: string | null
  businessName: string | null
  createdAt: string
}

export type CreateAdminUserRequest = {
  email: string
  firstName: string
  lastName: string
  password: string
  businessId: string
}

File: src/Pasukhi.Domain/Entities/AdminUser.cs
using Microsoft.AspNetCore.Identity;

namespace Pasukhi.Domain.Entities;

public class AdminUser : IdentityUser
{
    public string FirstName { get; set; } = string.Empty;
    public string LastName { get; set; } = string.Empty;
    public Guid? BusinessId { get; set; }
    public Business? Business { get; set; }
    public DateTime CreatedAt { get; set; }
    public string? ExternalProvider { get; set; }
    public string? ExternalProviderId { get; set; }
}
